language-plutus-core/exe/Main.hs

```diff
@@ -54,16 +62,55 @@ stdInput = flag' StdInput
   (  long "stdin"
   <> help "Read from stdin" )
 
+
+data Output = FileOutput FilePath | StdOutput
+
+output :: Parser Output
+output = fileOutput <|> stdOutput
+
+fileOutput :: Parser Output
+fileOutput = FileOutput <$> strOption
+  (  long "output"
+  <> short 'o'
+  <> metavar "FILENAME"
+  <> help "Output file" )
+
+stdOutput :: Parser Output
+stdOutput = flag' StdOutput
+  (  long "stdout"
+  <> help "Write to stdout" )
+
+
+data Format = Plc | Cbor  -- Input/output format for programs
+
+format :: Parser Format
+format = flag Plc Cbor
+  ( long "cbor"
+  <> long "CBOR"
```

Comment:
Two long flags again.

Reply:
But it seems rather convenient that both `cbor` and `CBOR` are allowed.

Reply:
Is it? what command line tools accept both capitalized and non-capitalized versions of flags?

Reply:
Dunno, maybe it's not.

Reply:
> Is it? what command line tools accept both capitalized and non-capitalized versions of flags?

`cp`, `rm`, `tr`, ...

Reply:
huh, TIL.

Reply:
What annoys me is that CBOR's usually written in upper case, so `--CBOR` is the obvious name for the flag, but then that's a bit of a pain to type.  That's why I added both upper and lower case versions.  In fact I'm tempted to double down and do the same for CK/CEK.